tools/demo/nativescript-google-login/index.ts

```diff
@@ -1,68 +1,58 @@
 import { DemoSharedBase } from '../utils';
-import { GoogleLogin } from '@imagene.me/nativescript-google-login';
-import { Application } from '@nativescript/core';
-import { Dialogs } from '@nativescript/core';
+import { GoogleLogin, GoogleLoginError } from '@imagene.me/nativescript-google-login';
+import { Application, isIOS } from '@nativescript/core';
 
 export class DemoSharedNativescriptGoogleLogin extends DemoSharedBase {
 	googleLogin: GoogleLogin;
 
 	init(): void {
 		try {
-			this.googleLogin = new GoogleLogin(
-				{
-					serverClientId: '1093279712153-m5ggq5i59cogt538c62fg3lss7tforig.apps.googleusercontent.com',
-				},
-				Application.android.foregroundActivity
-			);
+		  if (isIOS) {
+        this.googleLogin = new GoogleLogin(
+          {
+            serverClientId: "1093279712153-m5ggq5i59cogt538c62fg3lss7tforig.apps.googleusercontent.com",
+            clientId: "1093279712153-qgrdtsrg76kleref0ip14k2jgpktq8h0.apps.googleusercontent.com",
+          },
+          Application.ios.rootController,
+        );
+      } else {
+        this.googleLogin = new GoogleLogin(
+          {
+            serverClientId: '1093279712153-m5ggq5i59cogt538c62fg3lss7tforig.apps.googleusercontent.com',
+          },
+          Application.android.foregroundActivity
+        );
+      }
 		} catch (error) {
 			console.log(error);
 		}
 	}
 
 	login(): void {
+    if(isIOS) {
+      this.googleLogin.setIosUIViewController(Application.ios.rootController);
```

Comment:
Jest sens to ustawiać skoro podajemy to do constructora?

Reply:
Ten root controller nie jest dostępny w onInit i afterViewInit dlatego dodałem te metody do ustawienia poza konstruktorem